### ISSUE TITLE
Integrated images are not rendered if there is no `id`

### ISSUE THREAD

**Issue by Massedil** [the-djmaze/snappymail#339]
**Describe the bug**
Integrated images are not rendered if there is no `id`

**To Reproduce**

1. Open the .eml file with snappymail and thunderbird

**Expected behavior**
Integrated images are displayed and not seen as attachment.

**Screenshots**
![test_snappymail](https://user-images.githubusercontent.com/20357783/164469415-5ac73f6b-c084-48c3-98ef-af4fd3ddba77.png)
![test_thunderbird](https://user-images.githubusercontent.com/20357783/164469419-91dfb2c6-2b33-4426-9e05-dc617c75d5e1.png)

**Desktop (please complete the following information):**
 - Browser firefox 99
 - SnappyMail Version 2.15.0

**Additional context**

HTML source : 
```
<!doctype html>
<html>
<head>
<title>Test text/html</title>
</head>
<body>
<h1>Test text/html</h1>
<p>Twitter :</p>
<img alt=3D"test" height=3D"auto" src=3D"cid:twitter.png" >
<p>Facebook :</p>
<img alt=3D"test" height=3D"auto" src="cid:facebook.png" >
<p>Oups :</p>
<img alt="oups.gouv.fr" id="oups" src="cid:oups.png">
<p>END</p>
</body>
</html>
```

EML : 

- [test.eml](https://github.com/the-djmaze/snappymail/files/8531469/test.txt)

**Comment by the-djmaze** [comment]
It's not about the `id` it's about invalid quoted-printable.

Something is wrong with the send/received HTML.
Which mail app is used to send the message?

**Comment by Massedil** [comment]
> It's not about the id, it's about invalid quoted-printable.

How can I send you privately the original .eml file because I have (wrongly ?) simplified a lot the .eml and the invalid quoted-printable is probably because of me.

> Which mail app is used to send the message?

It is an email from impots.gouv.fr.

**Comment by Massedil** [comment]
In the original .eml, I have this : 

```
=09=09=09=09=09=09=09=09=09=09=09=09=09<img alt=3D"oups.gouv.fr" id=3D"oups=
" src=3D"cid:oups.png">

[...]

=09=09=09=09=09=09=09=09=09=09=09=09=09=09=09=09=09<img alt=3D"" src=3D"cid=
:twitter.png" style=3D"border:0;display:block;outline:none;text-decoration:=
none;width:100%;" width=3D"45" height=3D"auto">

[...]

=09=09=09=09=09=09=09=09=09=09=09=09=09=09=09=09=09<img alt=3D"" src=3D"cid=
:facebook.png" style=3D"border:0;display:block;outline:none;text-decoration=
:none;width:100%;" width=3D"45" height=3D"auto">
```

I don't know if it is sufficient to reproduce the issue.

**Comment by the-djmaze** [comment]
> I don't know if it is sufficient to reproduce the issue.

No it's not. You can send it to Massedil-issue-339@snappymail.eu

**Comment by Massedil** [comment]
I sent you the email, let me know if I can help you more.

**Comment by the-djmaze** [comment]
I've tested your mail and it works here as expected.
So i have no clue why yours is failing
![afbeelding](https://user-images.githubusercontent.com/3752035/165412977-18167bce-c169-4921-b2da-7796b355ed23.png)

**Comment by Massedil** [comment]
I first thought it can be linked with my Firefox extensions, so I removed all and tried again, but no, same behavior :

![snappymail](https://user-images.githubusercontent.com/20357783/165950920-103cbc9e-03be-4904-b331-b97c3c702947.png)

> So i have no clue why yours is failing

I don't understand how it is possible that my render differs from yours... 

I use Firefox 99.0 from Ubuntu 20.04. 
I tried on Firefox for Android with my smartphone and I have the same behavior.
I use snappymail 2.15.0 release (so not the github master branch version).

To be sure, here is a checksum of the .eml file I use and sent to you : 

```
md5sum snappymail_339.eml 
7cacd873771bd72693d128cdbd0c6bfc  snappymail_339.eml
```

Here is my "About" details, maybe we have some php extensions that differs and causes the rendering diff : 

![snappymail2](https://user-images.githubusercontent.com/20357783/165954295-e3c6c3e9-c4c7-4981-9792-0cc67b06b78f.png)

The strange thing is that, more than the fact images are not all displayed, snappymail show them as attachment, and not on your side.

Other point I have seen that differ on you screenshot and mine : 
- you have some text in white and for me it is black here :  "Découvrez-le sur le site [aquoiserventmesimpots.gouv.fr](https://www.aquoiserventmesimpots.gouv.fr/).".
- The block part "Pour votre sécurité :" disappeared on your screenshot.

Thanks again, and let me know I can help you more.
